internal/market/coin.go

```diff
@@ -29,6 +32,8 @@ type VolatileCoins map[string]VolatileCoin
 
 type CoinMap map[string]Coin
 
+type CoinVolumeTradedMap map[string]float64
```

Comment:
`market.TradeVolumes` is a little easier to read and grasp than `market.CoinVolumeTradedMap`:

```suggestion
type TradeVolumes map[string]float64
```

I can tell you picked this variable based on `CoinMap`, if you have a better name for that one be my guest. Generally I try to avoid naming things after the thing. Like the type is a map, so you don't need to include the word 'map' in it.

Reply:
Exactly, I was trying to adapt to your codebase. God bless Goland's refactoring features. Maybe `ExchangeCoins`, if you also want to change `CoinMap`? You can also keep it like this and maybe change it later tbh.

Reply:
Yeah well maybe just `market.Coins`, not too long and can't really mistake that for something else imo.